EIPS/eip-7642.md

```diff
@@ -0,0 +1,104 @@
+---
+eip: 7642
+title: eth/69 - Drop pre-merge fields
+description: Drop unnecessary fields after the merge
+author: Marius van der Wijden (@MariusVanDerWijden)
+discussions-to: https://ethereum-magicians.org/t/eth-70-drop-pre-merge-fields-from-eth-protocol/19005
+status: Draft
+type: Standards Track
+category: Networking
+created: 2024-02-29
+requires: 5793
+---
+
+## Abstract
+
+After the merge a few fields (`td`) and messages (`NewBlockHashes`, `NewBlock`) in the networking protocol became obsolete.
+This EIP modifies the networking messages such that these fields are not sent anymore.
+Additionally we propose to remove the `Bloom` field from the receipts networking messages.
+
+## Motivation
+
+We recently discovered that none of the clients store the `Bloom` field of the receipts as it can be recomputed on demand.
+However the networking spec requires the `Bloom` field to be send over the network.
```

Comment:
```suggestion
However the networking spec requires the `Bloom` field to be sent over the network.
```

Reply:
@ mods theres someone typo-farming on my PR

Reply:
they have been removed from the face of the earth